styles.css

```diff
@@ -538,31 +531,207 @@ h1.heading {
   margin-bottom: 8px;
 }
 
-@media screen and (max-width: 768px) {
-  .container_small {
-    padding-left: 8px;
-    margin-top: -60px;
+.hand {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    background-color: rgb(240, 240, 240);
+  }
```

Comment:
- [ ] Since the background color for both the mobile and desktop versions is the same, you should not have the `body` tag here.

Reply:
thanks man 
done